### DISCUSSION TITLE
Tiered Unlock Sounds

### DISCUSSION THREAD

**Discussion by LiquifiedSnow** [RetroAchievements/RAIntegration#921] [Ideas]
Requesting the ability to set custom unlock sounds based off points value of unlocked achievement.
If unlock.wav is present it'll use that for all but if you'd like custom ones something like

1-5 Points unlock1.wav
10 Points unlock2.wav
25 Points unlock3.wav
50 Points unlock4.wav
100 Points unlock5.wav

**Comment by StingX2** [comment]
Do we have a mastery unlock?

**Comment by Jamiras** [comment]
https://github.com/RetroAchievements/RAWeb/discussions/747 discusses reporting the unlock rate. Unique sounds could be associated to that too (which would probably contradict this). I know at one point someone suggested having a sound for "rare" achievements. XBOX has a unique chime for things earned by less than 10% of players.
